package.json

```diff
@@ -55,14 +55,15 @@
         "nx": "^14.5.10",
         "postcss": "^8.3.6",
         "prettier": "^2.1.2",
-        "react-test-renderer": "^16.13.1",
+        "react-test-renderer": "^17.0.2",
         "stylelint": "^13.13.1",
         "stylelint-config-prettier": "^8.0.2",
         "stylelint-formatter-pretty": "^2.1.1",
         "stylelint-no-unsupported-browser-features": "^4.0.0",
         "stylelint-selector-bem-pattern": "^2.1.0",
         "stylelint-webpack-plugin": "^2.1.1",
-        "ts-jest": "^26.4.2"
+        "ts-jest": "^26.4.2",
+        "webpack-cli": "^4.7.2"
```

Comment:
Why we need to add this in root package.json since we already have it in every package?

Reply:
Not sure why but I got errors saying that webpack-cli` is not installed. will check it again.

Reply:
removed it @mahdiyeh-fs . thanks